### DISCUSSION TITLE
Engine model and winds at cruise level

### DISCUSSION THREAD

**Discussion by CptBayer** [flybywiresim/aircraft#6192] [General]
Hi developers,

short question about the engine model. I guess the development of the model is still in progress? Cause I observed, that the fuel consumption with tail wind is higher than on the picture with more or less head wind .. I had tailwinds with round about 80 kts and a fuel burn of 1.100 kg/hr each ...

![grafik](https://user-images.githubusercontent.com/76531893/142406738-698af8ec-2783-402a-ba2f-0117c83c55e8.png)

greets

**Comment by tracernz** [comment]
Still in progress. Free time is often hard to come by. ;)
